compute_endpoint/tests/unit/test_boot_persistence.py

```diff
@@ -32,14 +32,7 @@ def fake_ep_dir(fs: fakefs.FakeFilesystem, ep_name) -> pathlib.Path:
 detach_endpoint: false
 display_name: null
 engine:
-    type: GlobusComputeEngine
-    address: 127.0.0.1
-    provider:
-        type: LocalProvider
-        init_blocks: 1
-        min_blocks: 0
-        max_blocks: 1
-        """.strip()
+    type: ThreadPoolEngine""".strip()
```

Comment:
```suggestion
    type: ThreadPoolEngine
        """.strip()
```
If you are doing formatting anyway, the offset """ is more confusing IMO.

Reply:
Heh; we have different eyes.  I disagree, but I also don't care.  I've moved it to below, and lined it up with the opening triple-quotes, in both this and the other comment thread.

Reply:
Interesting - probably because I prefer to see the string itself more clearly separate from the .strip(), and you think the .strip() is part of the """ and so should be together.  Far away from a tab vs. spaces POV difference 🤣